SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -636,7 +639,7 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
                            su2double val_omega_Inf, const CConfig* config)
       : CNumerics(val_nDim, 2, config),
         idx(val_nDim, config->GetnSpecies()),
-        sustaining_terms(config->GetKind_Turb_Model() == TURB_MODEL::SST_SUST),
+        sustaining_terms(false),      //TODO this is temporary
```

Comment:
could this be replaced with the structure? or would we template the source terms?